INZFS.MVC/Views/QuestionDescription/BusinessProposal/_BusinessUploadRiskRegister.cshtml

```diff
@@ -10,5 +10,5 @@
               </ul>
 
 
-              <p class="govuk-body">You can <a href="#" class="govuk-link govuk-!-font-weight-regular">download a risk register template</a> or use your own.</p>
+              <p class="govuk-body">You can <a href="../media/EEF9 Q5.17. Risk register template.xls" download class="govuk-link govuk-!-font-weight-regular">download a risk register template</a> or use your own.</p>
```

Comment:
I recommend using tilde (root operator).

Example : href="~/media/EEF9 Q5.8. Cost and performance pathway template.xlsx"

Reply:
resolved please review